packages/aws-cdk-lib/aws-logs/lib/log-group.ts

```diff
@@ -524,6 +524,11 @@ export interface SubscriptionFilterOptions {
    * Log events matching this pattern will be sent to the destination.
    */
   readonly filterPattern: IFilterPattern;
+
+  /**
+   * The name of the subscription filter.
+   */
+  readonly filterName?: string;
```

Comment:
need a default here

Reply:
Thank you for your comment! Sorry, I don't understand your intention. What do you mean `need a default`? As I described in the below conversation, I think it's better that filterName is optional because this can still let CDK/CFn generate physical id (filter name) automatically when this prop isn't specified.

Reply:
For optional properties, we supply a default value in the documentation so our users know what it means when they don't supply anything:

```ts
/**
 * The name of the subscription filter.
 *
 * @default - a generated name is created for you
 */
readonly filterName?: string;
```

Reply:
Makes sense! I aligned with logGroupName and logStreamName.

https://github.com/aws/aws-cdk/blob/e53e8dcede7b2ffc84f8f5151eff9ace3b005684/packages/aws-cdk-lib/aws-logs/lib/log-group.ts#L381-L386
https://github.com/aws/aws-cdk/blob/e53e8dcede7b2ffc84f8f5151eff9ace3b005684/packages/aws-cdk-lib/aws-logs/lib/log-group.ts#L502-L509